src/components/App/App.js

```diff
@@ -0,0 +1,28 @@
+require('normalize.css/normalize.css')
+require('./App.css')
+
+import React from 'react'
+import Navbar from '../Navbar/Navbar'
```

Comment:
I moved it to components because there are 6 folders in src now so I was thinking maybe is better to limit it for readability especially if this one is used quite rarely. But yes, now I see it wasn't best idea and anyway this project folders structure need some improvement ;)